src/IssueViz.UnitTests/Editor/Common/CommonTaggerProviderTestsBase.cs

```diff
@@ -46,7 +45,7 @@ public void CreateTagger_BufferIsNull_Throws()
         [TestMethod]
         public void CreateTagger_BufferIsNotNull_ReturnsSingletonTagger()
         {
-            var buffer = CreateValidTextBuffer();
+            var buffer = TaggerTestHelper.CreateBuffer();
```

Comment:
Is there a reason not have a `using static` like the other tests that use the `TaggerTestHelper`?

Reply:
I feel dumb, I didn't know this feature exists!